govcd/api_vcd_test.go

```diff
@@ -63,8 +64,12 @@ type TestConfig struct {
 		SysOrg   string `yaml:"sysOrg"`
 	}
 	VCD struct {
-		Org     string `yaml:"org"`
-		Vdc     string `yaml:"vdc"`
+		Org         string `yaml:"org"`
+		Vdc         string `yaml:"vdc"`
+		ProviderVdc struct {
+			Name           string `yaml:"name"`
```

Comment:
We need to add at least a network pool to create an usable VDC.
We could either add a specific network pool to the test fields, or search for the available pools.
I don't know if we can easily use a network pool with the API at our disposal. 
What do you think?

Reply:
I don't mind adding another parameter here for the network pool.  Is it ok to follow the same pattern using the query api to run a lookup by name?